src/environments/environment.prod.ts

```diff
@@ -1,6 +1,10 @@
 export const environment = {
   production: true,
-  config: {
-    apiHost: 'mindseyesociety.org/services'
+
+  externalUrls: {
+    authLogin: 'https://api-stage.mindseyesociety.org/users/v1/auth/signin/ux-dev',
+    authLogout: 'https://portal.mindseyesociety.org/logout',
```

Comment:
@AerionDyseti - Should call `https://api-stage.mindseyesociety.org/users/v1/auth/signout` to properly destroy token before redirecting to the portal.